taskt/UI/Forms/Supplement Forms/frmDisplayManager.cs

```diff
@@ -84,19 +86,19 @@ private void tmrCheck_Tick(object sender, EventArgs e)
                     }
 
 
-                    LogEvent("Machine '" + machine.MachineName + "' is due for desktop login");          
+                    LogEvent("Machine '" + machine.MachineName + "' is due for desktop login");
                     machine.LastKnownStatus = "Attempting to login";
                     machine.NextConnectionDue = DateTime.Now.AddMinutes(2);
                     LogEvent("Next Connection for Machine '" + machine.MachineName + "' due at '" + machine.NextConnectionDue + "'");
-                    Supplemental.frmRemoteDesktopViewer viewer = new Supplemental.frmRemoteDesktopViewer(machine.MachineName, machine.UserName, machine.Password, windowWidth, windowHeight, chkHideScreen.Checked, chkStartMinimized.Checked);
+                    frmRemoteDesktopViewer viewer = new frmRemoteDesktopViewer(machine.MachineName, machine.UserName, machine.Password, windowWidth, windowHeight, chkHideScreen.Checked, chkStartMinimized.Checked);
```

Comment:
Can you shorten this line?

Reply:
I can't find this. It's already broken up in my version

Reply:
Actually, it was done by me.
You could get it reduced after merge.

Reply:
got it. okay